client/verta/verta/_deployment/endpoint.py

```diff
@@ -172,3 +172,19 @@ def get_status(self):
         _utils.raise_for_http_error(response)
         return response.json()
 
+    def get_access_token(self):
+        url = "{}://{}/api/v1/deployment/workspace/{}/endpoints/{}/stages/{}/accesstokens".format(
+            self._conn.scheme,
+            self._conn.socket,
+            self.workspace,
+            self.id,
+            self._get_or_create_stage(),
+        )
+        response = _utils.make_request("GET", url, self._conn)
+        _utils.raise_for_http_error(response)
+        data = response.json()
+        tokens = data["tokens"]
+        if len(tokens) == 0:
+            return None
+        return tokens[0]
```

Comment:
Needs to return the token itself as str
```suggestion
        return tokens[0]['creator_request']['value']
```